ansible/dualtor/nic_simulator/nic_simulator.py

```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/env python3
 """
-_   _ _____ _____    _____ _____ __  __ _    _ _            _______ ____  _____  
-| \ | |_   _/ ____|  / ____|_   _|  \/  | |  | | |        /\|__   __/ __ \|  __ \ 
-|  \| | | || |      | (___   | | | \  / | |  | | |       /  \  | | | |  | | |__) |
-| . ` | | || |       \___ \  | | | |\/| | |  | | |      / /\ \ | | | |  | |  _  / 
-| |\  |_| || |____   ____) |_| |_| |  | | |__| | |____ / ____ \| | | |__| | | \ \ 
-|_| \_|_____\_____| |_____/|_____|_|  |_|\____/|______/_/    \_\_|  \____/|_|  \_\
+_   _ _____ _____    _____ _____ __  __ _    _ _            _______ ____  _____         # noqa W605
+| \ | |_   _/ ____|  / ____|_   _|  \/  | |  | | |        /\|__   __/ __ \|  __ \       # noqa W605
+|  \| | | || |      | (___   | | | \  / | |  | | |       /  \  | | | |  | | |__) |      # noqa W605
+| . ` | | || |       \___ \  | | | |\/| | |  | | |      / /\ \ | | | |  | |  _  /       # noqa W605
+| |\  |_| || |____   ____) |_| |_| |  | | |__| | |____ / ____ \| | | |__| | | \ \       # noqa W605
+|_| \_|_____\_____| |_____/|_____|_|  |_|\____/|______/_/    \_\_|  \____/|_|  \_\      # noqa W605
```

Comment:
Can you change this docstring to just a comment to avoid adding `# noqa W605`?

Reply:
OK, just changed